index.html

```diff
@@ -1,10 +1,58 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="mdl-js">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Registration</title>
+    <link rel="stylesheet" href="visual.css">
+    <link rel="shortcut icon" href="src/short_icon.png" type="image/png">
 </head>
+
 <body>
-<h1>Day is gone!</h1>
+<<<<<<< HEAD
+<img class="displayed" src="src/short_icon.png" width="90px"/>
+=======
+<img class="displayed" src="src/short_icon.png" />
+>>>>>>> full form
+<div class="css-form">
+    <p class="heading-with-line">Форма регистрации</p>
+    <div class="form">
+        <form id="form_id" action="" method="post" onsubmit="return validate();">
```

Comment:
Use `Event` binding methods in js file, instead of inline-html evnts like `onsubmit` etc.

Reply:
Example:
`object.addEventListener("submit", myScript);`